tests/MSBuildDeviceIntegration/Resources/LinkDescTest/HttpClientTest.cs

```diff
@@ -4,16 +4,16 @@
 public class HttpClientTest
 {
 	// [Test]
-	public static string NewHttpClient()
+	public static string GetString()
```

Comment:
Now that we know the issue in https://github.com/dotnet/runtime/issues/67402, it occurs to me that we should also have a "Post" or "Put" test as well. That way we ensure writing to the stream works, it takes a different enough code path that we should ensure it works.

See https://github.com/dotnet/runtime/issues/67402#issuecomment-1090305928

Reply:
I changed this to do a post and read the response back.